src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -91,9 +136,18 @@ const AcademiaSignUpForm = React.memo(({ setSuccess, success, ...props }) => {
             institution_name: institutionName,
             institution_type: institutionType,
             agree_to_email: agreeToEmail,
+            instructor_type:
+                instructorType === 'other'
+                    ? instructorTypeDesc
+                    : instructorType,
+            instructor_interests: instructorInterest,
+            country: country,
+            course_name: courseName,
+            course_syllabus: courseSyllabus,
```

Comment:
I think we are good to merge this so long as in Realm we add these to `onAcademiaRegistration` where the email is sent to the Academia team